index.md

```diff
@@ -1,7 +1,10 @@
 ---
 title: "Make your Data Management Plan comes true - Tips and Tricks"
-summary: Practical data management guidelines for Life Sciences researchers.
 ---
 
 
+DM Hub is an information resource about research data management for biologists and scientists in Life Sciences, curated by [ELIXIR Belgium](https://www.elixir-belgium.org). It aims to communicate essential best practices and practical guidelines that researchers can apply to manage their data, from the beginning to the end of the research project. The recommended best practices and guidelines are accepted by several scientific communities across Europe and data management experts in [ELIXIR](elixir).
```

Comment:
"DM Hub" in this text should be replace it with "Data Management Hub (DM Hub)". We need to introduce the acronymous before using it.